packages/gestalt/src/TableRowDrawer.js

```diff
@@ -0,0 +1,67 @@
+// @flow strict
+import { type Node, Children, cloneElement, Fragment, useEffect, useRef, useState } from 'react';
+import cx from 'classnames';
+import styles from './Table.css';
+import Box from './Box.js';
+import { useTableContext } from './contexts/TableContext.js';
+
+type Props = {|
+  /**
+   * Must be instances of Table.Cell. See the [Subcomponent section](https://gestalt.pinterest.systems/web/table#Subcomponents) to learn more.
+   */
+  children: Node,
+  /**
+   * The contents within the drawer.
+   */
+  drawerContents: Node,
+  /**
+   * Unique id for Table.RowDrawer.
+   */
+  id: string,
+|};
+
+/**
+ * Use [Table.RowDrawer](https://gestalt.pinterest.systems/web/table#Table.RowDrawer) to define a row drawer to display additional content.
+ */
+export default function TableRowDrawer({ children, drawerContents, id }: Props): Node {
+  const { stickyColumns } = useTableContext();
+  const rowRef = useRef();
+  const [columnWidths, setColumnWidths] = useState([]);
+
+  const cs = cx(styles.drawer);
```

Comment:
This isn't necessary — you can pass `styles.drawer` directly where `cs` is used

Reply:
Didnt know this.
DONE